docs/instrumentation_examples.md

```diff
@@ -0,0 +1,214 @@
+# Armory Instrumentation Examples: Measuring Experiment Artifacts Using Probes and Meters
+For an introduction to `Probe`s and `Meter`s, please refer to [Measurement Overview](./metrics.md).
+
+## Context
+To monitor particular aspects of an `armory run` session, the user needs to know the following factors:
+- What am I measuring?
+- When should I measure it?
+- Where should my custom monitoring script go?
+
+We assume the user is capturing artifacts from the model or attack and wishes to use `Probe`s and `Meter`s to monitor certain variables within the code.
+
+Recall the steps for a minimal working example (in [Measurement Overview](./metrics.md#instrumentation)):
+1. Create `Probe` via `get_probe(name)`
+2. Place `Probe` actions
+3. Create `Meter` with processing functions that take input from created `Probe`
+4. Connect `Meter` to `Hub` via `get_hub().connect_meter(meter)`
+
+The examples will show how each of these steps are accomplished.
+
+## Example 1: Measuring a Model Layer's Output
+### User Story
+I am interested in layer output from the second `relu` activation of a `forward` method located in `armory/baseline_models/pytorch/cifar.py`.
```

Comment:
`in layer output` --> `in the layer output`

Reply:
resolved with c24837a7